scalability_test/create_notebook.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ $# -ne 3 ]; then
+    echo "Usage: $0 <num_notebooks> <batch_size> <username>"
+    exit 1
+fi
+
+# Number of notebbooks to create
+num_notebooks=$1
+# Number of requests to make concurrently
+batch_size=$2
+username=$3
+SECONDS=0
+
+create_resources() {
+    notebook_name=$1
+    # Create service account
+    sed "s/ope-placeholder/$notebook_name/g" sa_template.yaml | oc apply -f -
+
+    # Create pvc
+    sed "s/ope-placeholder/$notebook_name/g" pvc_template.yaml | oc apply -f -
+
+    # Create notebook
+    sed -e "s/ope-placeholder/$notebook_name/g" -e "s/your_username/$username/g" notebook_template.yaml | oc apply -f -
+}
+
+for ((i=0; i<$num_notebooks; i+=batch_size)); do
+    for ((j=0; j<$batch_size && (i+j)<$num_notebooks; j++)); do
+        notebook_name="ope-scalability-$((i+j))"
```

Comment:
These resource names won't be unique between different test runs, which will result in conflicts if you start a new test before cleaning up an old test, or if two people run tests at the same time. We should include something like a random hash in all names.